### ISSUE TITLE
Very slow container data loading

### ISSUE THREAD

**Issue by safaci2000** [cockroachdb/cockroach#131877]
**Describe the problem**

I'm curious to know why the data set I'm loading is so much slower.  I'm trying to initialize cockroachdb using very similar dataset and letting the docker container load the .SQLs.

I'm not sure if this is cockroachdb issues or some bad choices made in how the docker setup works.

The load time for postgres is: around 12 seconds.  The load time for cockroach DB is around 1 min and 20 seconds.  Give or take.

I'm trying to understand if this is expected, or I'm doing something silly.

**To Reproduce**

Initialize a docker-compose or docker stack as you like with the data provided here: https://github.com/go-jet/jet-test-data/tree/master/init/cockroach

container parameters: "start-single-node", "--accept-sql-without-tls" 
tested against the latest release:  cockroachdb/cockroach:v23.1.27

**Expected behavior**

Load time should be relatively comparable to what postgres or mysql is for a similar data set.

load time for Mysql/Maria is around 7-8 seconds, Postgres 12-13 seconds, Cockroachdb DB is well at the minute mark.  Sqlite is well, local disk IO, so ignoring that.

Excluding all of my own code, and only looking at the docker container data loading, it's still at the minute mark just to load data.

**Environment:**
* CockroachDB version used: v23.1.27
* Browser name and version: NA
* Operating system and version: Mac OS X M2 chipset

Jira issue: CRDB-42732

**Comment by alyshanjahani-crl** [comment]
This issue was discussed in community slack: https://cockroachdb.slack.com/archives/CP4D9LD5F/p1728053066398149

TLDR: Use of individual `insert into` statements on a single node can lead to poor performance, prefer using `IMPORT`
